python_on_whales/utils.py

```diff
@@ -315,6 +316,25 @@ def format_time_for_docker(time_object: Union[datetime, timedelta]) -> str:
         return f"{time_object.total_seconds()}s"
 
 
+def format_signal_arg(signal_object: Optional[Union[int, str]]) -> Optional[str]:
+    if signal_object is None:
+        return None
+    else:
+        return format_signal_for_docker(signal_object)
+
+
+def format_signal_for_docker(signal_object: Union[int, str]) -> str:
+    if isinstance(signal_object, int):
+        return signal.Signals(signal_object).name
+    elif isinstance(signal_object, str):
+        if hasattr(signal.Signals, "SIG" + signal_object):
+            return "SIG" + signal_object
+        else:
+            return signal_object
+    else:
+        raise TypeError(f"Got unexpected signal type {type(signal_object).__name__!r}")
```

Comment:
Fine by me. Next time, maybe let's use quotes directly for simple stuff, since `!r` is not something I expect python newcomers to understand. But obviously I'm not going to nitpick on this beautiful PR

Reply:
In general I prefer `!r` since it will call `__repr__()`, which is not necessarily the same as just putting single quotes around it (and works for things other than strings too - it's a useful thing to know!).
```
In [2]: print(repr("say 'hello'!"))
"say 'hello'!"
```